util/crypto/crypto.go

```diff
@@ -0,0 +1,57 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"errors"
+	"io"
+)
+
+func passphraseToKey(passphrase string) []byte {
+	hasher := sha256.New()
+	hasher.Write([]byte(passphrase))
+	key := hasher.Sum(nil)
+	return key
+}
+
+// Encrypt encrypts the given data with the given passphrase.
+func Encrypt(data []byte, passphrase string) ([]byte, error) {
+	block, err := aes.NewCipher(passphraseToKey(passphrase))
+	if err != nil {
+		return nil, err
+	}
+	gcm, err := cipher.NewGCM(block)
+	if err != nil {
+		return nil, err
+	}
+	nonce := make([]byte, gcm.NonceSize())
+	if _, err = io.ReadFull(rand.Reader, nonce); err != nil {
+		panic(err.Error())
+	}
+	ciphertext := gcm.Seal(nonce, nonce, data, nil)
```

Comment:
What's the role of the `nonce` as first param here?

Reply:
Ah nevermind, I see that the ciphertext gets appended to the nonce, so you have access to the nonce in Decrypt.